bin/PerlDDS/Run_Test.pm

```diff
@@ -464,7 +464,132 @@ sub wait_kill {
 
   my $process = $self->{processes}->{process}->{$name}->{process};
 
-  return PerlDDS::wait_kill($process, $wait_time, $name, $verbose, $opts);
+  my $result = PerlDDS::wait_kill($process, $wait_time, $name, $verbose, $opts);
+
+  # If opts has indicated that user wants to get core dump/signal information
+  # then find the core dump file if any and print its stack trace.
+  if (defined $opts && defined $opts->{dump_ref} && ${$opts->{dump_ref}}) {
+      # Get the core file pattern
+      my $core_pattern_file = "/proc/sys/kernel/core_pattern";
+      if (!(-e $core_pattern_file)) {
```

Comment:
I assume this just fails on Windows (which is expected / fine)? What about MacOS? Does it use this file as well, same syntax?

Reply:
The mechanism on macOS seems different; I think it doesn't have a core file pattern. I'll need to find out more about macOS and maybe Windows.

Reply:
I added support for macOS. It's in the PR to the ACE repo. The GitHub Actions workflow file will need updates to make it able to generate core files. I'll work on this later.

Reply:
Should we make a follow-up Jira task to fix GitHub Actions for rmacOS?

Reply:
Yes, we can do that. I'm still not able to make core files generated for macOS on GitHub Actions even though the same commands worked locally. I think it's better to fix this later, so it doesn't hold up the rest of the PR.